build/codegen/codegen.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S bash -e
+#!/usr/bin/env bash
```

Comment:
should we keep -e

Reply:
This is covered by `set -o errexit` now, later in the script